src/index.html

```diff
@@ -7,13 +7,90 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Dia</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&family=Poppins:wght@400;500;600;700&family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
   <body>
-    <h1>Dia</h1>
+    <header class="header">
+      <nav class="navigation header__navigation">
+        <a
+          href="#logo"
+          class="navigation__item navigation__item--logo"
+        >
+          <img
+            src="./images/logo/dia-logo.svg"
+            alt="logo"
+          />
+        </a>
+
+        <a
+          href="#menu"
+          class="navigation__item navigation__item--menu"
+        >
+          <img
+            src="./images/Vector.svg"
+            alt="menu"
+          />
+        </a>
```

Comment:
for desktop you should replace burger menu with links (you can make use of display: none)